README.md

```diff
@@ -27,8 +27,11 @@ If you're interested in learning more about our governance or becoming a member
 
 ## Join the fun!
 
-We have a meeting every other Wednesday at **19:00 UTC**.  This is 13:00 PST (UTC-7), 16:00 US EST (UTC-4), 20:00 BST (UTC+1), and 04:00 on Thursday JST (UTC+9): <a target="_blank" href="https://calendar.google.com/event?action=TEMPLATE&amp;tmeid=MWgxZnN2cWR2NjBhZGlhODBwNXRoc3RrMDBfMjAxOTA3MTBUMTkwMDAwWiAxM3JhM3R1ZXFrcWJmZnBmMmc1NmZvMmN0c0Bn&amp;tmsrc=13ra3tueqkqbffpf2g56fo2cts%40group.calendar.google.com"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>    
-We also have a [discord channel][discord].
+* We have a [Google Hangouts](https://hangouts.google.com/call/BgkpIXkZghZH92NqjNWsAEEI) meeting every other Wednesday at **19:00 UTC**.
```

Comment:
I noticed you deleted the google calendar link and I think it's useful for people to be able to add it to their calendar. I like the reminders. Maybe add the hangouts link separately if you really want ?

Reply:
Ah, sure. I can fix that. The date was old so I thought the Hangouts part of it was the only important part left.